src/convertors/logseq/logseq-convertor.ts

```diff
@@ -30,7 +30,13 @@ export class LogseqConvertor extends Convertor {
       {},
     )
 
-    const convertedNotes = validated.blocks.map((note) => this.convertLogseqNote(note))
+    // Convert all notes except for whiteboard pages
+    const convertedNotes = validated.blocks.flatMap((note) => {
+      if (note?.properties?.['ls-type'] === 'whiteboard-page') {
```

Comment:
Let's have a helper function:

export function isWhiteboardPage(value: LogseqBlock): asserts value is LogseqWhiteboardBlock {
  ...
}

Reply:
I implemented the first suggestion and implemented something like this second suggestion but not quite right here.  The empty blocks are happening in the children of whiteboard pages.  So I'm still doing this check here to completely skip whiteboard pages and then much later I'm asserting that child blocks are of the right type.  

We could remove this check and build an empty Reflect page for whiteboard pages.  Or could give them a fixed body like "The body of this page could not be imported...".  

Also, this is my first time using assertion functions.  Thanks for pointing it out.

Reply:
Sweet, almost there, left one more comment.